### PR TITLE
test: simple delegation regression tests

### PR DESCRIPTION
**Pull request by BATMAH69**

### CODE REVIEW

File: tests/regression/test_voting_acc.py [hunk 1]
```diff
@@ -121,3 +121,99 @@ def test_can_object(stranger, test_vote):
...
+#  holder 1 reset delegate
+# stranger revote for YES holders 1-3 (reject)
+# stranger revote for YES holders 1 (reject not a delegate)
+# stranger revote for YES holders 2 (reject not a delegate)
+# stranger revote for YES holders 3 (reject cant overpower holder)
+# holders 1,2,3 can revote main phase YES
+# holders 1,2,3 can revote objection phase NO
+"""
+
+def test_attempt_vote_for_multi(call_target, stranger, test_vote):
```
Comment (Psirex) [inline tests/regression/test_voting_acc.py:144]: Maybe title it something like "test_delegation" or "test_simple_delegation"?

File: tests/regression/test_voting_acc.py [hunk 2]
```diff
@@ -121,3 +121,99 @@ def test_can_object(stranger, test_vote):
...
+    holder3 = LDO_VOTE_EXECUTORS_FOR_TESTS[3]
+
+    assert contracts.voting.getVotePhase(vote_id) == 0  # Main phase
+    # stranger vote for YES holders 1,2 (reject no delegation)
+    with reverts("REJECT"):
+        contracts.voting.attemptVoteForMultiple(vote_id, True, [holder1, holder2], {"from": stranger})
+
+    # holders 0,1 delegate to stranger
+    for holder in [holder1, holder2]:
+        contracts.voting.setDelegate(stranger, {"from": accounts.at(holder, force=True)})
```
Comment (Psirex) [inline tests/regression/test_voting_acc.py:158]: Semantically, it's better to introduce a standalone fixture, "delegate," cause a stranger usually means "anybody"/"entity without required rights".

File: tests/regression/test_voting_acc.py [hunk 3]
```diff
@@ -121,3 +121,99 @@ def test_can_object(stranger, test_vote):
...
+        contracts.voting.attemptVoteForMultiple(vote_id, True, [holder3], {"from": stranger})
+
+    # holders 1,2,3 can revote main phase YES
+    for holder in [holder1, holder2, holder3]:
+        contracts.voting.vote(vote_id, True, False, {"from": accounts.at(holder, force=True)})
+
+    chain.sleep(contracts.voting.voteTime() - contracts.voting.objectionPhaseTime())
+    chain.mine()
+
+    assert contracts.voting.getVotePhase(vote_id) == 1  # Objection phase
```
Comment (Psirex) [inline tests/regression/test_voting_acc.py:215]: Let's also add the check that the delegate can't vote "yea" during the objection phase.

File: tests/regression/test_voting_acc.py (hunk 2 above)
Comment (BATMAH69) [reply tests/regression/test_voting_acc.py:158]: @Psirex I am not sure that we have any required for delegates at the moment, but it will add mock.

**Pull request merged**